clients/src/test/java/org/apache/kafka/clients/FetchSessionHandlerTest.java

```diff
@@ -320,7 +315,7 @@ public void testDoubleBuild() {
         try {
             builder.build();
             fail("Expected calling build twice to fail.");
-        } catch (Throwable t) {
+        } catch (NullPointerException npe) {
```

Comment:
NPE is a weird way of enforcing idempotency for build()! While you are in this code base could we add a javadoc to build() to clarify this expectation?

Reply:
You're right! I've pushed an update with a few changes to improve this.

Reply:
Thank you. Looks good to merge!